assets/app/actions/dispatchActions.js

```diff
@@ -0,0 +1,72 @@
+import { dispatch } from '../store';
```

Comment:
The "actions" in siteActions, for instance, do much more than simply dispatch actions to the redux store. They also make multiple calls to github or the federalist APIs.

As part of refactoring after writing some tests, these "create an action and dispatch it to the redux store" functions emerged as a related cohesive set of logic. Pulling them out separated the business logic of the various API calls from the state changes spurred by dispatching actions to the redux store. Now the business logic siteActions do not need to understand what it means to create an action and dispatch it to the redux store.

The inconsistency comes from my not following through and performing the same transformations to the other actions (alertActions, buildActions, ...). That's confusing. The logical separation is between the non-redux-state-transforming logic and the redux-state-transforming bits.

Looking at where the code says it wants to go...